src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -455,8 +455,8 @@
   <data name="PropertiesFilesAndFoldersCountString" xml:space="preserve">
     <value>{0, number} {0, plural, one {file} other {files}}, {1, number} {1, plural, one {folder} other {folders}}</value>
   </data>
-  <data name="PropertiesLocationsCountString" xml:space="preserve">
-    <value>from {0, number} {0, plural, one {location} other {locations}}</value>
+  <data name="PropertiesFilesAndFoldersAndLocationsCountString" xml:space="preserve">
```

Comment:
i guess we can wemove `String` suffix.

Reply:
this will all be gradually adjusted, I don't know whether to change now